MAINTAINERS.md

```diff
@@ -7,7 +7,9 @@
 * `documentation`
   * `prometheus-mixin`: @beorn7
 * `storage`
-  * `remote`: @csmarchbanks, @cstyan
+  * `remote`
```

Comment:
This is organised by directory, so we shouldn't be going more granularly than that.

Reply:
Is there a reason it needs to be that way? There's nothing stopping me from splitting remote read and write into their own subdirectories within `storage/remote`. 

Listing people as maintainers of either remote read or remote write (or both) is more useful to those outside the project than just listing all three of us as maintainers of `storage/remote` when I personally am not going to be of any use to someone looking for a review on a remote read PR.

Reply:
That's the way I laid it out as it made sense to me. Any change to one likely affects the other, and we can route amongst ourselves anyway.